lib/msal-react/typedoc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "out": "./ref",
+    "excludePrivate": true,
+    "excludeProtected": true
+}
```

Comment:
👀 didnt know about this file. Would be good to update the other libs (can be done in separate PR).

Reply:
Made some changes to where and how typedocs are generated. We can discuss in our sync tomorrow.